src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this and why is it empty?

Reply:
It was added when I generate the project. I'm not sure what is it for but I will read about it and delete it if it's not necessary.